scripts/issue/689/settle_voter_positions.py

```diff
@@ -0,0 +1,95 @@
+from pycoingecko import CoinGeckoAPI
+
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import r
+from brownie import Contract
+
+
+PROD = False
+
+# thresholds
+MAX_BAL_DEPOSIT = 12_000 * 1e18
+MAX_WBTC_SELL = 1.5
+
+vault = GreatApeSafe(r.badger_wallets.treasury_vault_multisig)
+voter = GreatApeSafe(r.badger_wallets.treasury_voter_multisig)
+bal = voter.contract(r.treasury_tokens.BAL)
+
+
+def mint_aurabal_and_lock_aura():
+    voter.init_balancer()
+    voter.init_aura()
+
+    wrapper = voter.contract(r.aura.wrapper)
+    aura = voter.contract(r.treasury_tokens.AURA)
+    aurabal = voter.contract(r.treasury_tokens.AURABAL)
+    vlaura = vault.contract(r.aura.vlAURA)
+
+    voter.take_snapshot([bal, aura, aurabal])
+
+    # mint auraBAL up to MAX_BAL_DEPOSIT
+    to_deposit = bal.balanceOf(voter)
+    if to_deposit > MAX_BAL_DEPOSIT:
+        to_deposit = MAX_BAL_DEPOSIT
+
+    bal.approve(wrapper,to_deposit)
+    wrapper.deposit(
+        to_deposit,  # uint256 _amount
+        wrapper.getMinOut(to_deposit, 9950),  # uint256 _minOut
+        False,  # bool _lock
+        r.aura.aurabal_staking  # address _stakeAddress
+    )
+
+    # transfer to vault to eventually deposit into bauraBAL
+    aurabal_amount = Contract(r.aura.aurabal_staking).balanceOf(voter)
```

Comment:
i think that amount will be zero, since nothing should be stake in behalf of the voter

```suggestion
    aurabal_amount = aurabal.balanceOf(voter)
```

Reply:
yup, `aurabal.balanceOf(voter)` shows correct after changing `_stakeAddress` to `ZERO_ADDRESS`